spatial-history/index.md

```diff
@@ -30,10 +19,42 @@ It makes sense to put an introductory paragraph here, above the map and buttons.
+
+According to Eirik the Red’s Saga it was on a voyage to Greenland from Norway that Leif Eriksson was swept off course and “lighted upon lands of which before he had no expectation” finding wheat, vine-trees, and maples.[^3] There is some dissonance between Eirik the Red’s Saga and The Saga of the Greenlanders in regards to the details of some of the voyages, but the two texts make it clear that following this initial discovery there were many more expeditions to North America, including failed attempts to settle. The descriptions of the resources available such as whales, game, and grapes as well as the topographical descriptions give credence to the idea that Vinland is in North America.[^4] Other areas described in the sagas seem to describe Baffin Island and Labrador, respectively.  
+
+It is actually the failed attempts at settlement described in the sagas that further make the case for Vinlands location because one of the key reasons that the Vikings didn’t stay is due to conflict with Indigenous populations. There is, again, some discrepancy about this in the sagas but it is definitely clear that the explorers encountered Indigenous populations. The Saga of the Greenlanders implies that the expedition fights with the indigenous peoples but then leaves for unrelated reasons[^5], while Eirik the Red’s states that “. . . despite everything the land had to offer there, they would be under constant threat of attack from its prior inhabitants.”[^6]
+
+# Archaeological Evidence
+Although the saga’s are a valuable source in understanding Viking culture and are good place to start if you want to understand the phenomenon of Vinland, they are not enough to empirically prove that Vikings came to North America. For years the archeological evidence unavailable and Vinland was dismissed as a work of fiction, until Helge Ingstad and Anne Stine Ingstad discovered the Norse settlements of L’anse aux Meadows in 1960. Interestingly, the settlements in L’anse aux Meadows do not include any barns or evidence of livestock (which would imply that it intended as a long-term settlement). Instead, it is made up of dwellings and workshops that suggest that it was used as an access point to the St. Lawrence, a stop over in which to take a break and fix your ship before venturing further south.[^7]
```

Comment:
some pictures & specifics here would be helpful. Scholarly sources may help. An [incredibly simple google scholar search](https://scholar-google-ca.myaccess.library.utoronto.ca/scholar?q=l%27anse-aux-meadows+&hl=en&as_sdt=0,5) turns up the original articles from the 70's; use date filtering to get access to more recent work.  More ocmplex search queries will of course turn up different results.